src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,10 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User userMail = new UserService().findByEmail(email);
```

Comment:
```suggestion
        User foundUser = new UserService().findByEmail(email);
```
let's use more informative variable names